.ember-cli

```diff
@@ -0,0 +1,9 @@
+{
+  /**
+    Ember CLI sends analytics information by default. The data is completely
+    anonymous, but there are times when you might want to disable this behavior.
+
+    Setting `disableAnalytics` to true will prevent any data from being sent.
+  */
+  "disableAnalytics": false
+}
```

Comment:
We don't, actually. Here's how ember-cli calculates `disableAnalytics`: https://github.com/ember-cli/ember-cli/blob/b915143cd2e4d2067cebcd4dc93c2bf4d5b98afa/lib/cli/index.js#L57-L61. And here's how Yam parses files: https://github.com/twokul/yam/blob/master/lib/config.js#L10-L25. So if the file is missing, Yam treats it as an empty Object, which means `config.get('disableAnalytics')` will be just as falsy as it is now.